packages/widget.vite-plugin/src/build-plugin/__tests__/buildWidgetSetManifest.test.ts

```diff
@@ -0,0 +1,236 @@
+/*
+ * Copyright 2025 Palantir Technologies, Inc. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import type { ParameterDefinition, WidgetSetInputSpec } from "@osdk/widget.api";
+import { describe, expect, test } from "vitest";
+import { buildWidgetSetManifest } from "../buildWidgetSetManifest.js";
+import type { WidgetBuildOutputs } from "../getWidgetBuildOutputs.js";
+
+describe("buildWidgetSetManifest", () => {
+  const WIDGET_SET_RID = "ri.foundry.main.widget-set.test-widget-set";
```

Comment:
if you want this to be more representative of the real world:
```suggestion
  const WIDGET_SET_RID = "ri.widgetregistry..widget-set.test";
```